checksit/cvs.py

```diff
@@ -25,26 +25,53 @@ def _load(self, vocab_id):
         vocab_file = os.path.join(vocabs_dir, f"{vocab_id}.json")
         self._vocabs[vocab_id] = json.load(open(vocab_file))
 
+    def _load_from_url_ncas(self, vocab_id_url):
```

Comment:
Throughout the repo there's currently a mix of docstrings or no docstrings - while I agree with both points, I think this could do with it's own dedicated PR to blitz through the whole repo